admission-webhook/integration_tests/templates/simple-with-gmsa.yml

```diff
@@ -16,12 +16,11 @@ spec:
     metadata:
       labels:
         app: {{ .TestName }}
-      annotations:
-        pod.alpha.windows.kubernetes.io/gmsa-credential-spec-name: {{ index .CredSpecNames 0 }}
     spec:
       serviceAccountName: {{ .ServiceAccountName }}
+      securityContext:
+        windowsOptions:
+          gmsaCredentialSpecName: {{ index .CredSpecNames 0 }}
       containers:
-      - image: nginx
+      - image: k8s.gcr.io/pause
```

Comment:
Any reason to use the pause image instead of `nginx` as it was before? The `name` for the container here and in other templates above seem to indicate `nginx`.